README.md

```diff
@@ -69,27 +71,225 @@ class Genre {
+lordOfTheRings;
+// => Book {
+//    title: 'The Lord of the Rings',
+//    author: Author { firstName: 'J.R.R.', lastName: 'Tolkien' },
+//    genre: Genre { name: 'Fantasy' },
+//    publishingDate: '1954' }
+```
+
+Another important benefit we gain is that we can have a `class` use its _own_
+property to access _another_ `class`es data:
+
+```js
+lordOfTheRings.author; // => Author { firstName: 'J.R.R.', lastName: 'Tolkien' }
+lordOfTheRings.author.lastName; // => 'Tolkien'
+```
+
+This is the most basic way `class`es can work together - by depending on each other
+to store related data.
+
+## Sharing Behavior between Multiple Classes
+
+In addition to data, `class`es can also utilize each other's methods. For
+instance, we could add a getter to our `Author` class called `fullname`:
+
+```js
+class Author {
+	constructor(firstName, lastName) {
+		this.firstName = firstName;
+		this.lastName = lastName;
+	}
+
+	get fullname() {
+		return `${this.firstName} ${this.lastName}`;
+	}
+}
+```
+
+...and then utilize this in `Book`:
+
+```js
+class Book {
+	constructor(title, author, genre, publishingDate) {
+		this.title = title;
+		this.author = author;
+		this.genre = genre;
+		this.publishingDate = publishingDate;
+	}
 
-## SWBAT 1
+	get infoStatement() {
+		return `${this.title} was written by ${
+			this.author.fullname
+		} and published in ${this.publishingDate}.`;
+	}
+}
+
+lordOfTheRings.infoStatement;
+// => 'The Lord of the Rings was written by J.R.R. Tolkien and published in 1954.'
+```
+
+Here, we're using properties of the `Book` `class` and `fullname` from the
+`Author` `class` together to form a sentence.
+
+## Sharing Data and Behaviors Creates Dependencies between Classes
+
+One thing to always keep in mind when working with multiple `class`es:
+If one `class` uses data or behaviors from another `class`, the first `class`
+becomes _dependent_ on the second.
+
+In our `Book` `class`, for instance, we've written `infoStatement` using
+`this.author.fullname`. The `infoStatement` getter is expecting there to be a
+`fullname` property (or pseudo-property) present in `this.author`. If we
+mistakenly created a `Book` instance using only `String`s again and then tried
+to use `infoStatement` we will have a problem:
+
+```js
+let unfinishedTales = new Book(
+	'Unfinished Tales',
+	'J.R.R. Tolkien',
+	'Fantasy',
+	'1980'
+);
+
+unfinishedTales;
+// => Book {
+//    title: 'Unfinished Tales',
+//    author: 'J.R.R. Tolkien',
+//    genre: 'Fantasy',
+//    publishingDate: '1954' }
+
+unfinishedTales.author;
+// => 'J.R.R. Tolkien'
+unfinishedTales.author.fullname;
+// => undefined
+unfinishedTales.infoStatement;
+// => 'Unfinished Tales was written by undefined and published in 1980.'
+```
+
+Because of what we have written, `Book` is has become dependent on `Author` in
+order to fully function correctly. If we make changes to the `Author` `class`,
+say we change the name of a property, we could potentially break functionality
+in `Book`.
```

Comment:
Maybe add some comments in the code snippet, or be more detailed about what was just shown in the code example. I am finding myself losing track of what's happening from the last example to this one.